pkg/utils/deployer.go

```diff
@@ -306,7 +319,7 @@ func resolveControllerRef(descLister applisters.DescriptionLister, namespace str
 }
 
 func ApplyDescription(ctx context.Context, clusternetClient *clusternetclientset.Clientset, dynamicClient dynamic.Interface,
-	discoveryRESTMapper meta.RESTMapper, desc *appsapi.Description, recorder record.EventRecorder) error {
+	discoveryRESTMapper meta.RESTMapper, desc *appsapi.Description, recorder record.EventRecorder, handler ResourceInformerHandler) error {
```

Comment:
Since we're using informers for GVKs to watch changes of every deployed resource, there is no need to still sync `Description` here, which would be tedious.

Reply:
Yes, I keep hub deployer not changed.